### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by alicecoleman**
Guide to browser and device testing

### CODE REVIEW

File: testing/device and browser.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+In testing phase we are required to be more thorough and test on laptop, tablet and mobile directly and over multiple devices, more on that below.
+
+# Browser testing
+
+There are numerous browsers available and we need to ensure Idearium projects function correctly on all the popular ones. Please test the following browsers and browser versions on each device;
+
+- Laptop/desktop
+  - Chrome (latest)
+  - Firefox (latest)
+  - IE11
```
Comment (smebberson) [inline testing/device and browser.md]: Do we need to test Edge, specifically?

File: testing/device and browser.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Tablet
+  - Safari (latest)
+  - Chrome (latest)
+
+- Mobile
+  - Chrome (latest)
```
Comment (smebberson) [inline testing/device and browser.md]: Should this list the mobile platforms, too? For example, Android and iOS.

File: testing/guide.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# When to test
+
+You will know if something is ready to test once it has been moved into the `QA` or `verify` column on the project board. 
+
+Once you have tested a story and verified it works you should move it into `code review` or `done`. Which columns are applicable will depends on the project you are working on just take a look at the project board to find out which ones you need.
+
+# How to test
```
Comment (smebberson) [inline testing/guide.md]: I think there should be a section in here on trying to intentionally break something. To try stupid things. To try things with the slow connection settings. To paste in stupid stuff. To post forms without completing them, etc.

File: testing/device and browser.md (hunk 1 above)
Comment (alicecoleman) [reply testing/device and browser.md]: Edge only makes up about 4% of users and they use the Chrome technology so it is always consistent with Chrome. I don't think we will worry.

File: testing/device and browser.md (hunk 2 above)
Comment (alicecoleman) [reply testing/device and browser.md]: I listed Android because that has more users on mobile than tablet. I thought testing tablet in safari would cover off on mobile but do you think it needs to be included in mobile too? Chrome is by far the most popular browser though.

File: testing/guide.md (hunk 3 above)
Comment (smebberson) [reply testing/guide.md]: @alicecoleman, could you please follow this up, I'd like to merge this in.

**Comment by allanchau** [comment]
@idearium/developers Can this be merged in yet?

File: testing/device and browser.md (hunk 1 above)
Comment (smebberson) [reply testing/device and browser.md]: @alicecoleman, I think we need to test Edge. It's the newest browser and that stat will change over time.

File: testing/device and browser.md (hunk 2 above)
Comment (smebberson) [reply testing/device and browser.md]: Yeah, I think Safari in mobile for sure.

**Comment by alicecoleman** [comment]
@smebberson this is done now.
